koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py

```diff
@@ -113,6 +113,9 @@ def update_summary_tables(self, start_date, end_date):
         with OCPReportDBAccessor(self._schema) as accessor:
             with schema_context(self._schema):
                 report_period = accessor.report_periods_for_provider_uuid(self._provider.uuid, start_date)
+                if not report_period:
+                    LOG.info(f"No report period for {self._provider.uuid} with start date {start_date}")
```

Comment:
**Question**

Should this be a warning? Should the log message also indicate that no processing has occurred?

Reply:
Should it be a `LOG.warning` instead of a `LOG.info`?

Reply:
Log info has been replaced with warning.